python-spec/src/somacore/data.py

```diff
@@ -153,14 +215,41 @@ def read(
         self,
         coords: options.DenseNDCoords,
```

Comment:
Is there a reason that we specify a default for `SparseNDArray.read` but not `DenseNDArray.read`?

Reply:
my guess is that was inadvertent

@bkmartinjr to verify

Reply:
Added the default (in the same separate commit).